easybuild/easyconfigs/x/XGBoost/XGBoost-1.5.0-foss-2021a.eb

```
@@ -0,0 +1,26 @@
+easyblock = 'PythonPackage'
+
+name = 'XGBoost'
+version = '1.5.0'
+
+homepage = 'https://github.com/dmlc/xgboost'
+description = """XGBoost is an optimized distributed gradient boosting library designed to be highly efficient,
+ flexible and portable."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['6b7c34a18474c1b73b5c6dcf1231f5fe102d6f26490a65465e3879048bf1f3d4']
+
+builddependencies = [('CMake', '3.20.1')]
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+]
+
+use_pip = True
+download_dep_fail = True
+sanity_pip_check = True
+
```

There is core count in `setup.py` that just gets all CPU cores in the system. It should be changed to get actual number of cores available to the process

```suggestion

preinstallopts = "sed -i 's/nproc = os.cpu_count.*$/nproc = %(parallel)s/' setup.py &&"

```